server/middleware/reactApplication/ssrDisabled.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { renderToStaticMarkup } from 'react-dom/server';
+import { useStaticRendering } from 'mobx-react';
+import Helmet from 'react-helmet';
+import sha256 from 'sha256';
+
+import ServerHTML from './ServerHTML';
+
+useStaticRendering(true);
+
+/**
+ * React application middleware without server-side rendering
+ */
+export default function reactApplicationMiddleware(request, response) {
+  // Add script hashes
+  // See the server/middleware/security.js for more info.
```

Comment:
I should probably refactor this to reuse the addHash function across both versions of the middleware....